pages/storefront/edit.tsx

```diff
@@ -296,7 +296,7 @@ export default function Edit() {
               </TabPane>
             </Tabs>
             <Row justify="end">
-              <Button disabled={submitting} loading={submitting}>
+              <Button disabled={submitting} loading={submitting} onClick={onSubmit}>
```

Comment:
I believe `onClick={onSubmit}` should be `htmlType="submit" to avoid bypassing the form validation